src/effects/effectslot.h

```diff
@@ -133,6 +133,8 @@ class EffectSlot : public QObject {
     EngineEffectChain* m_pEngineEffectChain;
     QList<EffectKnobParameterSlotPointer> m_knobParameterSlots;
     QList<EffectButtonParameterSlotPointer> m_buttonParameterSlots;
+    QList<int> m_knobParameterPositionToSlot;
+    QList<int> m_buttonParameterPositionToSlot;
```

Comment:
These names do not make it clear what the "position" referred to is. Also, I think it is backwards; a map of slot indices to manifest indices would be more helpful for passing that information to the parameter slot.